config/crd/bases/e2etest.grpc.io_loadtests.yaml

```diff
@@ -8,6 +8,13 @@ metadata:
   creationTimestamp: null
   name: loadtests.e2etest.grpc.io
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .metadata.creationTimestamp
```

Comment:
Pods display "Age" last, so perhaps we should have State before Age for the load tests as well.

Generally speaking, I am more interested in whether a test has an error than whether it is 9h or 10h old.

Reply:
I am curious because the additionalPrinterColumns still show Age first. Is it going to display Age last, with the PR as it is?